lib/src/data/models/switch_data.dart

```diff
@@ -0,0 +1,36 @@
+enum SwitchPosition {
+  left,
+  right,
+}
+
+class SwitchData {
+  final double horizontalOffset;
+  final bool isDragging;
+
+  final _trackWidth = 35.0;
+  final _knobWidth = 15.0;
```

Comment:
UI stuff in the model... Hmmmmmmmmmmmmmmmm... I don't remember if I did this because separating it properly was too complex or because I just used easy mode during development.

Reply:
Denk dat jij dit best refactored aangezien je beter kan analyseren waarom je dit hebt gedaan 🧞